### PR TITLE
Precision fixes

### PR DESCRIPTION
**Pull request by Zhigal**
Hi, I made some important changes I use in my project, could be helpful imho...

### CODE REVIEW

File: leaflet/templates/leaflet/widget.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
     function {{ id_map_callback }}(map, options) {
         {{ module }}.store_class = {{ field_store_class }};
-        (new {{ geometry_field_class}}({{ module }})).addTo(map);
+        (new {{ geometry_field_class}}({...{{ module }}, precision: options.djoptions.precision})).addTo(map);
```
Comment (Gagaro) [inline leaflet/templates/leaflet/widget.html:20]: Please don't use spreading as it is not supported in IE and older edge version.

File: leaflet/static/leaflet/leaflet.css [hunk 2]
```diff
@@ -637,4 +637,12 @@
 /* Resize the "display_raw" textbox */
 .django-leaflet-raw-textarea {
 	width: 100%;
+}
+
+.leaflet-editing-icon {
```
Comment (Gagaro) [inline leaflet/static/leaflet/leaflet.css:642]: What does this have to do in this PR?

Comment (Zhigal) [reply leaflet/static/leaflet/leaflet.css:638]: I used it for debugging process.
Has been removed.

File: leaflet/templates/leaflet/widget.html (hunk 1 above)
Comment (Zhigal) [reply leaflet/templates/leaflet/widget.html]: Fixed

**Comment by Zhigal** [comment]
Hi all, made new changes according to the code review

File: leaflet/static/leaflet/leaflet.css (hunk 2 above)
Comment (Gagaro) [reply leaflet/static/leaflet/leaflet.css:638]: You remove `.django-leaflet-raw-textarea` and let `.leaflet-editing-icon`.

Comment (Zhigal) [reply leaflet/static/leaflet/leaflet.css:638]: Yes, it changes the shape of editing markets from a rectangle to a circle, which is more beautiful and elegant in my own opinion, do you want me to remove it?

Comment (Zhigal) [reply leaflet/static/leaflet/leaflet.css:638]: Yes, it changes the shape of editing markets from a rectangle to a circle, which is more beautiful and elegant in my own opinion, do you want me to remove it?

Comment (Gagaro) [reply leaflet/static/leaflet/leaflet.css:638]: Yes please, interface customization should be done in your project.

**Comment by Zhigal** [comment]
Wanted to change the shape of the editing marker, circle is more beautiful imho.
Removed those changes and use default rectangle markers.

File: leaflet/static/leaflet/leaflet.css [hunk 3]
```diff
@@ -633,8 +633,3 @@
 	margin-left: -12px;
 	border-right-color: #fff;
 	}
-
-/* Resize the "display_raw" textbox */
-.django-leaflet-raw-textarea {
-	width: 100%;
```
Comment (Gagaro) [inline leaflet/static/leaflet/leaflet.css:639]: Why did you also remove that part?

**Comment by Gagaro** [comment]
Could you also rebase on master for the CI, thanks.

File: leaflet/static/leaflet/leaflet.css (hunk 3 above)
Comment (Zhigal) [reply leaflet/static/leaflet/leaflet.css:639]: Yes, that was removed as well. 
Why you do not want to try circle markers? It looks much better

**Comment by Zhigal** [comment]
> Could you also rebase on master for the CI, thanks.
Sorry, have no idea what it means, any suggestions where to look?

**Comment by Gagaro** [comment]
There are new commits on the master branch since you started the PR. Travis seems to have trouble with a change. So if you could rebase on master (https://git-scm.com/book/en/v2/Git-Branching-Rebasing) your PR would have all the last commits.

Also for the CSS, styling is not in the scope of this module. We aren't the only ones using it and changing the style for hundreds/thousands of sites is not something we should do.

**Comment by Zhigal** [comment]
Done, rebased these changes to master brunch and created new PR

**Pull request closed without merging**